sigh/lib/assets/resign.sh

```diff
@@ -639,6 +639,71 @@ function resign {
         # Start with using what comes in provisioning profile entitlements before patching
         cp -f "$PROFILE_ENTITLEMENTS" "$PATCHED_ENTITLEMENTS"
 
+        log "Removing blacklisted keys from patched profile"
```

Comment:
This change definitely matches steps 1 and 2:

1. Extract entitlements from provisioning profile
1. Remove blacklisted keys from profile entitlements

Reply:
Looks like you've also got latest updated blacklist from Buck 👍